conf/homeserver.yaml

```diff
@@ -1271,9 +1274,7 @@ enable_registration: __ENABLE_REGISTRATION__
 
 # The user must provide all of the below types of 3PID when registering.
 #
-registrations_require_3pid:
- - email
-# - msisdn
+_REGISTRATION_REQUIRE_3PID_SED_PARAM_
```

Comment:
Not sure to understand why this isnt a double `_` like for `__TURN_SERVER_CONFIG__` to avoid calling sed manually ? x_X

Reply:
Well because we need many workaround for this...
cf #449